### ISSUE TITLE
SelectManyMenu: Improved Keyboard Support

### ISSUE THREAD

**Issue by weiprd** [primefaces/primefaces#7820]
**Describe the defect**
In the showcase 'input' of the Diamond theme it is not possible to reach the corresponding checkboxes at 'Listbox' by only using the keyboard.  It does not fulfill the WCAG 2.1 success criteria [2.1.1](https://www.w3.org/TR/WCAG21/#keyboard).
I guess the problem is not specific to this theme as it occures at the Rain theme too.

**Environment:**
 - PF Version: 10.0.0
 - Affected browsers: Chrome, Firefox, Edge

**To Reproduce**
1. Go to https://www.primefaces.org/diamond/input.xhtml
2. Set the focus in the search field of the Listbox 
3. Press tab key to go to the following checkbox
4. The whole section has the focus now. The checkboxes cannot be selected by only using the keyboard. Same behavior if you use the search function and only one result is displayed.  

**Expected behavior**
I would expect the following checkbox to get the focus and that it can be selected by using the space bar.

**Comment by melloware** [comment]
It's not Listbox its actually SelectManyMenu: https://www.primefaces.org/showcase/ui/input/manyMenu.xhtml?jfwid=9f10b

And its only an issue in Advanced rendering mode.

**Comment by melloware** [comment]
Duplicate of: https://github.com/primefaces/primefaces/issues/1435
